src/System.Text.Json/src/System/Text/Json/Serialization/JsonExtensionDataAttribute.cs

```diff
@@ -10,6 +10,13 @@ namespace System.Text.Json.Serialization
     /// </summary>
     /// <remarks>
     /// The TKey value must be <see cref="string"/> and TValue must be <see cref="JsonElement"/> or <see cref="object"/>.
+    /// 
+    /// During deserializing, when using <see cref="object"/> a "null" JSON value is treated as a null object reference, and when using
+    /// <see cref="JsonElement"/> "null" is treated as a JsonElement with <see cref="JsonElement.ValueKind"/> set to <see cref="JsonValueKind.Null"/>.
+    /// 
+    /// During serializing, the name of the extension data property is not included in the JSON; the properties are serialized as if
+    /// they were not missing.
+    /// 
```

Comment:
suggestion: `the properties are serialized as if they were not missing` -> `the data contained within the extension data is serialized as properties of the JSON object`.

Reply:
Done. Thanks